WMF Framework/Significant Events Endpoint/ArticleAsLivingDocViewModels.swift

```diff
@@ -944,7 +944,9 @@ public extension ArticleAsLivingDocViewModel.Event.Large {
         default:
             localizedString = String.localizedStringWithFormat(CommonStrings.manySectionsDescription, sections.count)
         }
-        
+
+        // Uncomment to remove extraneous HTML in the modal
+        // return " " + localizedString.removingHTML
```

Comment:
@staykids Actually I think it would work best if this was the only line you had. Generating the content insert html snippet above eventually calls this, so we would be stripping html **before** wrapping them in italic tags for the insert, thus getting to keep that styling (win-win). And I agree with you that it would be nice to remove that cruft from the modal descriptions. It would be good to test the View Discussion > talk page transition with this though. We should confirm that still works since we're relying on section titles to make that fragment.

Reply:
Cool - makes sense if we're dropping it both places. Updated. 

I don't know if there's a particular talk page case you've noticed, but so far I haven't noticed any behavior out of the ordinary when using the same test methodology I used in https://github.com/wikimedia/wikipedia-ios/pull/3769, using the Joe Biden and COVID-19 pandemic articles as test articles. Lemme know if you noticed a particular case or anything else specific to the view discussion > talk page transition you want me to test with this update.

Since our bad data is gone from the endpoint, I've been stubbing in our known bad data (from the [Phab ticket](https://phabricator.wikimedia.org/T268201#6634654)) as appropriate to make sure the result is intended. Still poking at this one, but wanted to go ahead and put up this update if you felt it was sufficient to approve/merge (as I don't wanna hold up any of your new RC plans.)